src/api/app/views/webui/project/show.html.erb

```diff
@@ -109,7 +109,11 @@
                 <%= link_to(sprited_text('tools-report-bug', 'Report bug'), bugzilla_url(@bugowners_mail, "#{@project.name}: Bug")) -%>
               </li>
           <% end -%>
-          <% if User.current.can_modify_project?(@project, true) %>
+          <% if @project.is_locked? && User.current.can_modify_project?(@project, true) %>
+                  <li>
+                    <%= link_to(sprited_text('lock_open', 'Unlock project'), { controller: 'project', action: 'unlock_dialog', project: @project.name }, remote: true) %>
+                  </li>
+          <% elsif User.current.can_modify_project?(@project) %>
```

Comment:
well, that is true. However, I like it this way anyway more, since I don't want to assume that there won't be further checks in future. And it runs only one can_modify_project check anyway at runtime (the reason why the project.is_locked? is before the can_modify_project check).

=> IMHO it is a better style to do the ignoreLock mode only for unlock button, but not for the other options.